PetAdoption-iOS/PetAdoption-iOS/Extensions/UIImage+Placeholder.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIImage+Placeholder.swift
+//  PetAdoption-iOS
+//
+//  Created by Keli'i Martin on 8/20/16.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import AlamofireImage
+
+extension UIImage
```

Comment:
Does the API framework actually download the images from the URLs and load them into image views?  Because I didn't see any of that in the framework...